### PR TITLE
Fastly Domains- Note

### PR DESCRIPTION
**Pull request by michellecolon-pantheon**
Closes # N/A

## Summary

**[AGCDN with WAF / IO Features](https://pantheon.io/docs/guides/agcdn/agcdn-wafio#efficient-image-optimization-at-the-edge)** - Add note about Fastly requiring perms to add domains.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by michellecolon-pantheon** [comment]
@eabquina Can you please take a look at the note I added re: Fastly domains?

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7574](https://pr-7574--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `52` | `52` | `0` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7574` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/40357/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/40357/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7574--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/guides/agcdn/03-agcdn-wafio.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ Support for XSS vulnerabilities.
 
 Image Optimization (IO) renders images using high-density points of presence (POPs) that cache content for longer times and serve millions of image variations. Serving from the edge in this way offloads work from your servers and improves your site's performance. IO supports several formats, including animated GIFs.
 
+<Alert title="Note"  type="info" >
+
+Pantheon partners with [Fastly](https://www.fastly.com/), if you prefer to use a CDN outside of the Pantheon platform. Fastly requires permissions to add domains to accounts, if it detects that a domain already exists.
```
Comment (eabquina) [inline source/content/guides/agcdn/03-agcdn-wafio.md:30]: This sounds like this is try to cover "Domain Delegation". 

Domain Delegation is a process where Fastly's platform detects if a domain is already used by another account. When it does, it blocks use of the domain and requires a "verification" process (usually via TXT records) for Fastly to allow use of a specific domain. 

However, it is only applicable to **Legacy AGCDN**. This won't be applicable to new SKUs moving forward.

So maybe we don't need to add this here, and place the topic of Domain Delegation somewhere else.

**Review by eabquina — CHANGES_REQUESTED** [review]

File: source/content/guides/agcdn/03-agcdn-wafio.md (hunk 1 above)
Comment (michellecolon-pantheon) [reply source/content/guides/agcdn/03-agcdn-wafio.md]: Hi @eabquina - I moved the alert to the Global CDN page, if you could take a look!

**Comment by stevector** [comment]
I don't think this PR is applicable given updates to AGCDN

**Pull request closed without merging**